dataflow-migrate-schedules/README.adoc

```diff
@@ -0,0 +1,110 @@
+= Cloud Scheduler Migration
+
```

Comment:
This documentation does not make it clear that the new SCDF server (the 2.3+ version) must be installed prior to the migration.  If you don't do that in advance, you get errors that the db has columns missing.

Reply:
Good catch.